.storybook/decorators/utilities.js

```diff
@@ -100,6 +131,12 @@ export const Container = ({
 			${when(heading, () => Heading({
 				...headingConfig,
 				content: heading
+			}, {
+				...context,
+				globals: {
+					...context.globals ?? {},
+					color: level === 1 ? "light" : context.globals?.color,
```

Comment:
Is a value of `1` for `level` always `"light"`? That association feels unclear when reading it without existing context.

Reply:
Generally but this is a good call-out because it's not the case here:  https://pr-3343--spectrum-css.netlify.app/preview/?path=/docs/components-action-button--docs&globals=color:dark

Reply:
This has now been fixed! cc @jawinn